Content.Shared/GameObjects/Components/Medical/SharedCloningPodComponent.cs

```diff
@@ -14,13 +14,23 @@ public class SharedCloningPodComponent : Component
         public class CloningPodBoundUserInterfaceState : BoundUserInterfaceState
         {
             public readonly Dictionary<int, string?> MindIdName;
-            public readonly float Progress;
+            // When this state was created.
+            public readonly TimeSpan ReferenceTime;
+            // Both of these are in seconds.
+            // They're not TimeSpans because of complicated reasons.
+            // CurTime of receipt is combined with Progress.
+            public readonly float Progress, Maximum;
+            // If true, cloning is progressing (predict clone progress)
+            public readonly bool Progressing;
             public readonly bool MindPresent;
 
-            public CloningPodBoundUserInterfaceState(Dictionary<int, string?> mindIdName, float progress, bool mindPresent)
+            public CloningPodBoundUserInterfaceState(Dictionary<int, string?> mindIdName, TimeSpan refTime, float progress, float maximum, bool progressing, bool mindPresent)
```

Comment:
Is there a reason why this isn't just sending start and end TimeSpans?

Reply:
It can't send start and end timespans because cloning can be interrupted. In this case, the start timespan would have to be weirdly warped. The logic makes more sense this way.